extensions/chromium/preferences_schema.json

```
@@ -81,6 +81,12 @@
       "description": "Whether to allow execution of active content (JavaScript) by PDF files.",
       "default": false
     },
+    "enableStampEditor": {
+      "title": "Enable stamp annotation editing",
+      "type": "boolean",
+      "description": "Whether to allow editing of stamp annotations.",
+      "default": true
+    },
```

Assuming this is temporary, I don't believe that we want to surface this in the Chrome-addon options-UI; hence you probably want the following.
```suggestion
    "enableStampEditor": {
      "type": "boolean",
      "default": true
    },
```